system/lib/update_compiler_rt.py

```diff
@@ -24,11 +24,9 @@ def main():
   shutil.rmtree(local_builtins)
   os.mkdir(local_builtins)
 
-  listfile = os.path.join(script_dir, 'compiler-rt', 'filelist.txt')
-  for name in open(listfile).read().splitlines():
-    src = os.path.join(upstream_src, name);
-    target = os.path.join(local_builtins, name);
-    shutil.copy2(src, target)
+  for pattern in ('*.c', '*.h', '*.inc'):
+    for src in glob.glob(os.path.join(upstream_src, pattern)):
+      shutil.copy2(src, local_builtins)
```

Comment:
It looks like this might be partially redundant with the next loop, which could be deleted.

Reply:
Filelist is still used to selectively build only a subset.  Building everything and removing the filelist is a followup to this PR.

Reply:
You are right.. I've got a followup that changes this again.. bit fixed the duplication here now